mkdocs.yml

```
@@ -81,6 +81,7 @@ nav:
           - 'Installation manual': variants/protectli_vp4620/installation-manual.md
           - 'Hardware Configuration Matrix': variants/protectli_vp4620/hardware-matrix.md
       - 'MSI PRO Z690-A WIFI DDR4':
+          - 'Building manual': variants/msi_z690/building.md
```

```suggestion
          - 'Building manual': variants/msi_z690/building-manual.md
```

I said the URL must be https://docs.dasharo.com/variants/msi_z690/building-manual/
Please change the file name accordingly

---

Fixed